README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## API Key
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+The planet API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
```

Comment:
Consider linking to the documentation for the CLI options. https://planetlabs.github.io/planet-client-python/cli/reference.html#general-options